### PR TITLE
New 'Multi Subjects' flag for all benchmark modes (also: don't delete the consumer if a consumer name is specified, fix consumer name argument name)

### PR DESCRIPTION
**Pull request by jnmoyne**

### CODE REVIEW

File: cli/bench_command.go [hunk 1]
```diff
@@ -667,13 +696,13 @@ func (c *benchCmd) runSubscriber(bm *bench.Benchmark, nc *nats.Conn, startwg *sy
 				progress.TimeStarted = startTime
 			}
 			if c.pull {
-				sub, err = js.PullSubscribe(c.subject, c.consumerName)
+				sub, err = js.PullSubscribe("getSubscribeSubject(c)", c.consumerName, nats.BindStream(c.streamName))
```
Comment (ripienaar) [inline cli/bench_command.go:699]: probably dont want the quotes here

**Review by ripienaar — CHANGES_REQUESTED** [review]
small tweak only

File: cli/bench_command.go (hunk 1 above)
Comment (jnmoyne) [reply cli/bench_command.go]: Ah yeah sorry I forgot to clean that up (that parameter is actually completely ignored (yup one of the many reasons we're redoing the JS API :)). Thanks for spotting this.

**Review by ripienaar — APPROVED** [review]
LGTM

**Pull request merged**